apidoc/ostree-sections.txt

```diff
@@ -694,3 +694,27 @@ ostree_kernel_args_from_string
 ostree_kernel_args_to_strv
 ostree_kernel_args_to_string
 </SECTION>
+
+<SECTION>
+<FILE>ostree-sign</FILE>
+OstreeSign
+OstreeSignDummy
+OstreeSignEd25519
+ostree_sign_list_names
+ostree_sign_commit
+ostree_sign_commit_verify
+ostree_sign_data
+ostree_sign_data_verify
+ostree_sign_get_by_name
+ostree_sign_get_name
+ostree_sign_add_pk
+ostree_sign_clear_keys
+ostree_sign_load_pk
+ostree_sign_set_pk
+ostree_sign_set_sk
+ostree_sign_summary
+<SUBSECTION Standard>
+ostree_sign_get_type
+ostree_sign_dummy_get_type
```

Comment:
Is this one just for our internal testing?  Seems odd to make it public API.

Reply:
Thank you!